flux-cli/src/test/java/com/marklogic/flux/impl/importdata/classifier/ImportAndClassifyFilesTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2025 MarkLogic Corporation. All Rights Reserved.
+ */
+package com.marklogic.flux.impl.importdata.classifier;
+
+import com.marklogic.flux.AbstractTest;
+import com.marklogic.junit5.XmlNode;
+import org.junit.jupiter.api.Test;
+
+class ImportAndClassifyFilesTest extends AbstractTest {
```

Comment:
Because we know testing against a real S4 will be difficult, just make an "options" test for now. Extend `AbstractOptionsTest` and look at some of the other subclasses to see how they work. You're just testing that the CLI options result in the correct Spark options being populated. 

That will likely suffice for this repo unless we get an easy way to test against S4.